packages/mui-material/src/Alert/Alert.js

```diff
@@ -61,10 +61,7 @@ const AlertRoot = styled(Paper, {
         [`& .${alertClasses.icon}`]: theme.vars
           ? { color: theme.vars.palette.Alert[`${color}IconColor`] }
```

Comment:
Will we also have to update CSS vars tokens in `extendTheme` to be the same in dark and light mode if CSS variables is used? I think we have to update it [here](https://github.com/mui/material-ui/blob/43a1b3d6e4e9063d17aef8692b4193fdc700e8f1/packages/mui-material/src/styles/experimental_extendTheme.js#L156-L159) to be `main`.

Reply:
Sweet, thanks for the review ⎯ it's done 👍